upstream-community-operators/istio/istio-operator.0.1.5.clusterserviceversion.yaml

```diff
@@ -0,0 +1,544 @@
+                        "name": "zipkin"
+                    }
+                ],
+                "includeIPRanges": "*",
+                "proxy": {},
+                "proxyInit": {},
+                "sidecarInjector": {
+                    "enabled": true,
+                    "initCNIConfiguration": {},
+                    "replicaCount": 1
+                }
+            }
+        }
+      ]
+spec:
+  displayName: Istio
+  description: |
+    # Istio-operator
+
+    Istio-operator is a Kubernetes operator to deploy and manage [Istio](https://istio.io/) resources for a Kubernetes cluster.
+
+    ## Overview
+
+    [Istio](https://istio.io/) is an open platform to connect, manage, and secure microservices and it is emerging as the `standard` for building service meshes on Kubernetes. It is built out on multiple components and a rather complex deployment scheme (around 14 Helm subcharts and 50+ CRDs). Installing, upgrading and operating these components requires deep understanding of Istio and Helm (the standard/supported way of deploying [Istio](https://istio.io/)).
+
+    The goal of the **Istio-operator** is to automate and simplify these and enable popular service mesh use cases (multi cluster federation, canary releases, resource reconciliation, etc) by introducing easy higher level abstractions.
+  keywords: ['istio', 'multi cluster', 'federation', 'service mesh', 'banzaicloud', 'open source']
+  version: 0.1.5
+  maturity: alpha
+  maintainers:
+  - name: Banzai Cloud
+    email: info@banzaicloud.com
+  provider:
+    name: Banzai Cloud
+  labels:
+    alm-owner-istio: istio-operator
+    operated-by: istio-operator
+  selector:
+    matchLabels:
+      alm-owner-istio: istio-operator
+      operated-by: istio-operator
+  links:
+  - name: Blog
+    url: https://banzaicloud.com/tags/istio
+  - name: Documentation
+    url: https://github.com/banzaicloud/istio-operator/blob/release-1.1/README.md
+  - name: Istio Operator Source Code
+    url: https://github.com/banzaicloud/istio-operator/tree/release-1.1
+
+  icon:
+  - base64data: PD94bWwgdmVyc2lvbj0iMS4wIiBlbmNvZGluZz0idXRmLTgiPz4KPHN2ZyB2ZXJzaW9uPSIxLjEiIHhtbG5zPSJodHRwOi8vd3d3LnczLm9yZy8yMDAwL3N2ZyIgdmlld0JveD0iMCAwIDMyMCAzMjAiPgogIDxnIGlkPSJsb2dvIiBmaWxsPSIjZmZmIj4KICAgIDxyZWN0IGlkPSJiYWNrZ3JvdW5kIiBmaWxsPSIjNDY2QkIwIiB3aWR0aD0iMzIwIiBoZWlnaHQ9IjMyMCIgLz4KICAgIDxwb2x5Z29uIGlkPSJodWxsIiBwb2ludHM9IjgwIDI1MCAyNDAgMjUwIDE0MCAyODAgODAgMjUwIi8+CiAgICA8cG9seWdvbiBpZD0ibWFpbnNhaWwiIHBvaW50cz0iODAgMjQwIDE0MCAyMzAgMTQwIDEyMCA4MCAyNDAiLz4KICAgIDxwb2x5Z29uIGlkPSJoZWFkc2FpbCIgcG9pbnRzPSIxNTAgMjMwIDI0MCAyNDAgMTUwIDQwIDE1MCAyMzAiLz4KICA8L2c+Cjwvc3ZnPgo=
+    mediatype: image/svg+xml
+  installModes:
+  - type: OwnNamespace
+    supported: true
+  - type: SingleNamespace
+    supported: false
+  - type: MultiNamespace
+    supported: false
+  - type: AllNamespaces
+    supported: true
+  install:
+    strategy: deployment
+    spec:
+      clusterPermissons:
+      - apiGroups:
```

Comment:
This item should include a `serviceAccountName` to match the permissions to

Reply:
Fixed, thanks @SamiSousa!